integration_tests/kpi.feature

```diff
@@ -53,8 +53,8 @@ Feature: KPI integration tests
      Given we have a scenario named kpi.d3asim_2103_nopv1
      And d3a is installed
      When we run the d3a simulation on console with kpi.d3asim_2103_nopv1 for 24 hrs (30, 30)
-     Then self_sufficiency of {'Grid': 1.0, 'Community': 0.666, 'House 1': 0.0, 'House 2': 0.5, 'House 3': 1.0} are correctly reported
-     And self_consumption of {'Grid': 1.0, 'Community': 0.8, 'House 1': None, 'House 2': 1.0, 'House 3': 0.5} are correctly reported
+     Then self_sufficiency of {'Grid': 1.0, 'Community': 0.833, 'House 1': 0.0, 'House 2': 0.5, 'House 3': 1.0} are correctly reported
```

Comment:
Whats the reason for change of KPI value in integration test?

Reply:
This was after a discussion with @spyrostz . We decided to move the expected outcome of the integrationtests again back to the original rationale behind this test: Test wether the Community used all its energy and sells the rest to the DSO (the same as in the corresponding 2PAB integrationtest).